fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckByline.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.LinkText
+import org.mozilla.fenix.compose.LinkTextState
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Review Quality Check byline with an embedded link to navigate to Fakespot.com.
+ *
+ * @param onLinkClick Invoked when the user clicks on the embedded link.
+ */
+@Composable
+@Suppress("LongParameterList")
+fun ReviewQualityCheckByline(
+    onLinkClick: (String) -> Unit,
+) {
+    LinkText(
+        text = stringResource(
+            id = R.string.review_quality_check_powered_by_link,
+            stringResource(
+                id = R.string.review_quality_check_powered_by,
+                stringResource(id = R.string.shopping_product_name),
+            ),
+        ),
+        linkTextState = LinkTextState(
+            text = stringResource(
+                id = R.string.review_quality_check_powered_by_link,
+                stringResource(id = R.string.shopping_product_name),
+            ),
+            url = "http://fakespot.com/",
```

Comment:
❓ Is this url confirmed? 
This should be extracted as a constant.

Reply:
There's a master string spreadsheet floating around with the links for the hyperlinks. (this one is in row 29)

I'll extract it though 👍

Reply:
got it, thanks!